hax/hax/util.py

```diff
@@ -449,6 +449,17 @@ def get_leader_session_no_wait(self) -> str:
             raise HAConsistencyException(
                 'Could not get the leader from Consul')
 
+    def is_leader_value_present_for_session(self) -> bool:
+        leader = self.kv.kv_get('leader')
+        if leader is None:
+            raise HAConsistencyException('No leader key exists yet')
```

Comment:
[Minor] maybe it makes sense to return False in this case?

Reply:
Yeah, I think so too, let's just return False instead of raising an exception.